Tasks/NuGetCommand/task.json

```diff
@@ -0,0 +1,447 @@
+            "label": "Use packages from this VSTS/TFS feed",
+            "defaultValue": "",
+            "helpMarkDown": "Include the selected feeds in the generated NuGet.config. You must have Package Management installed and licensed to select a feed here.",
+            "required": "false",
+            "groupName": "restoreAuth",
+            "visibleRule": "selectOrConfig = select",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "includeNuGetOrg",
+            "type": "boolean",
+            "label": "Use packages from NuGet.org",
+            "defaultValue": "true",
+            "helpMarkDown": "Include NuGet.org in the generated NuGet.config.",
+            "required": "false",
+            "groupName": "restoreAuth",
+            "visibleRule": "selectOrConfig = select"
+        },
+        {
+            "name": "nugetConfigPath",
+            "type": "filePath",
+            "label": "Path to NuGet.config",
+            "defaultValue": "",
+            "helpMarkDown": "The NuGet.config in your repository that specifies the feeds from which to restore packages.",
+            "required": "false",
+            "groupName": "restoreAuth",
+            "visibleRule": "selectOrConfig = config"
+        },
+        {
+            "name": "externalEndpoints",
+            "type": "connectedService:ExternalNuGetFeed",
+            "label": "Credentials for feeds outside this account/collection",
+            "required": false,
+            "helpMarkDown": "Credentials to use for external registries located in the selected NuGet.config. For feeds in this account/collection, leave this blank; the build’s credentials are used automatically.",
+            "properties": {
+                "EditableOptions": "False",
+                "MultiSelect": "True"
+            },
+            "groupName": "restoreAuth",
+            "visibleRule": "selectOrConfig = config"
+        },
+        {
+            "name": "noCache",
+            "type": "boolean",
+            "label": "Disable local cache",
+            "defaultValue": "false",
+            "helpMarkDown": "Prevents NuGet from using packages from local machine caches.",
+            "required": "false",
+            "groupName": "restoreAdvanced"
+        },
+        {
+            "name": "packagesDirectory",
+            "type": "string",
+            "label": "Destination directory",
+            "defaultValue": "",
+            "helpMarkDown": "Specifies the folder in which packages are installed. If no folder is specified, packages are restored into a packages/ folder alongside the selected solution, packages.config, or project.json.",
+            "required": "false",
+            "groupName": "restoreAdvanced"
+        },
+        {
+            "name": "verbosityRestore",
+            "type": "pickList",
+            "label": "Verbosity",
+            "defaultValue": "Detailed",
```

Comment:
fair enough

---
In reply to: [118801403](https://github.com/Microsoft/vsts-tasks/pull/4397#discussion_r118801403) [](ancestors = 118801403)